Sources/FoundationEssentials/String/String+Path.swift

```diff
@@ -373,6 +392,24 @@ extension String {
     
     func _resolvingSymlinksInPath() -> String? {
         guard !isEmpty else { return nil }
+
+#if os(Windows)
+        return self.withNTPathRepresentation {
+            let hFile: HANDLE = CreateFileW($0, GENERIC_READ, FILE_SHARE_READ | FILE_SHARE_WRITE | FILE_SHARE_DELETE, nil, OPEN_EXISTING, FILE_FLAG_BACKUP_SEMANTICS, nil)
```

Comment:
Just to make sure I have the right understanding, will `CreateFileW` actually create the file on disk if it does not exist, or is it just creating a handle to a file that may or may not exist?

Reply:
This particular call will _not_ create the file. The disposition (`OPEN_EXISTING`) controls the behaviour here.